tests/integration_tests/db_engine_specs/presto_tests.py

```diff
@@ -524,7 +524,7 @@ def test_query_cost_formatter(self):
         expected = [
             {
                 "Output count": "904 M rows",
-                "Output size": "354 GB",
+                "Output size": "329 GiB",
```

Comment:
These two values are identical except for the units. I took care not to change any of the other outputs.